etna/insights/migrations/0033_auto_20220304_2226.py

```diff
@@ -0,0 +1,411 @@
+            tag(name="Pay and pensions", slug="Pay and pensions"),
+            tag(name="Performing arts", slug="Performing arts"),
+            tag(name="Personal and family papers", slug="Personal and family papers"),
+            tag(name="Photography and film", slug="Photography and film"),
+            tag(name="Piracy and privateering", slug="Piracy and privateering"),
+            tag(
+                name="Planning (Land and Property)", slug="Planning (Land and Property)"
+            ),
+            tag(name="Polar", slug="Polar"),
+            tag(name="Policing", slug="Policing"),
+            tag(name="Population", slug="Population"),
+            tag(name="Poverty", slug="Poverty"),
+            tag(name="Transport management", slug="Transport management"),
+            tag(name="Travel and tourism", slug="Travel and tourism"),
+            tag(name="Treason and rebellion", slug="Treason and rebellion"),
+            tag(name="Transportation", slug="Transportation"),
+            tag(name="Trade and commerce", slug="Trade and commerce"),
+            tag(name="Tithes", slug="Tithes"),
+            tag(name="Taxation", slug="Taxation"),
+            tag(name="Sports", slug="Sports"),
+            tag(name="Slavery", slug="Slavery"),
+            tag(name="Shipping", slug="Shipping"),
+            tag(name="Sex and gender", slug="Sex and gender"),
+            tag(name="Sewerage", slug="Sewerage"),
+            tag(name="Royalty", slug="Royalty"),
+            tag(name="Royal Parks", slug="Royal Parks"),
+            tag(name="Road transport", slug="Road transport"),
+            tag(name="Resources", slug="Resources"),
+            tag(name="Research", slug="Research"),
+            tag(name="Renewable energies", slug="Renewable energies"),
+            tag(
+                name="Religious discrimination and persecution",
+                slug="Religious discrimination and persecution",
+            ),
+            tag(name="Religions", slug="Religions"),
+            tag(name="Refugees", slug="Refugees"),
+            tag(name="Rationing", slug="Rationing"),
+            tag(name="Railways", slug="Railways"),
+            tag(name="Radio and television", slug="Radio and television"),
+            tag(name="Race relations", slug="Race relations"),
+            tag(name="Public disorder", slug="Public disorder"),
+            tag(name="Treaties and alliances", slug="Treaties and alliances"),
+            tag(name="Weapons", slug="Weapons"),
+            tag(name="Witchcraft", slug="Witchcraft"),
+            tag(name="Votes for women", slug="Votes for women"),
+            tag(name="UFOs", slug="UFOs"),
+        ]
+    )
+
+
+def reverse_func(apps, schema_editor):
+    # forwards_func() creates tags instances,
+    # so reverse_func() should delete them.
+    tag = apps.get_model("taggit", "Tag")
+    db_alias = schema_editor.connection.alias
+    tag.objects.using(db_alias).filter(name="Africa", slug="Africa").delete()
```

Comment:
If these tags were added to a separate model, all you would need to do in the reverse function is:
```
InsightsTag.objects.all().delete()
```

Reply:
Yes works nicely :)